docs/Sprint-Review/sprint-25-summary.md

```diff
@@ -0,0 +1,70 @@
+- The UX Research team focused on Parsing Error blockers and understanding Regional Staff user journeys, and began investigations into the user access request journey
+- NextGen XMS and ACF AMS teams have given us technical information in order to assess and estimate our technical implimentation and recommendation to TDP Product Owner and tech lead. Our next task is to write integration tickets and estimate this work and provide a recommendation to OFA.
+
+
+## Sprint 25 Goals
+- Finish closing ATO functionality tickets
+- Focus on Secret Key Leakage Mitigation Epic
+- UX Ideation for Parsing Blocker Communications
+
+**Next Sprint (26) Goals:** 
+- Finish up Secret Key Leakage Mitigation Present recommendation re: authentication
+- TDP Permissions Epics (TDP Permissions Matrix, OFA Admins, PIV/CAC)
+- Continue work on Production readiness
+- UX Test Error Communication for Parsing Blockers
+
+## Merged/Completed (Done/Demo, Closed)
+- Django Admin a11y Fixes (Sprint 1) [#973](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/973)
+- (UX Sprint 24) Current State Analysis of Error Communication and Regional Staff Workflow Validation [#1018](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1018)
+
+## Submitted (QASP Review, OCIO Review)
+- Dependabot Mass Merge / Improvements [#1023](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1023)
+- Verify Admin Permissions Hierarchy and Roles [#1058](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1058)
+
+## Moving to Next Sprint (Raft Review, In Progress, Current Sprint Backlog)
+**Raft Review**
+- [EPIC] As an OFA admin, I can download raw file [#89](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/89)
+- Groups: Rename `Data Prepper` to `Data Analyst` [#1071](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1071)
+- Perform scheduled OWASP scans against deployed site(s) [#1032](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1032)
+- [Devops] Allow pa11y to scan views that require authorization [#1044](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1044)
+- [Frontend] Hook upload and download to real API endpoints [#834](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/834)
+
+
+**Blocked**
+- Update ATO docs and corresponding code docs [#962](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/962)
+- SPIKE: File transfer options for Tribal MVP [#1011](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1011)
```

Comment:
is this still blocked?

Reply:
oh good call. w/ access I think this can be closed?

Reply:
I had it removed from blocked. @lfrohlich  asked me to add it back in. Where do you want it represented?

Reply:
I think this should reflect what is blocked as of the end of the sprint. so if things are unblocked they should be moved in the zenhub board and then wouldn't need to be reflected here. Happy to discuss sequence/timing of sprint summary. Realizing we'll also want to update this at some point: https://github.com/HHS/TANF-app/blob/main/docs/How-We-Work/team-charter/sprint-schedule.md